packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -603,4 +603,6 @@ export enum UsageFlags {
   Error = 1 << 7,
   // Set when model is used in conjunction with an application/json content type.
   Json = 1 << 8,
+  // Set when model is used in conjunction with an application/xml content type.
+  Xml = 1 << 8,
```

Comment:
Shouldn't this be `1 << 9`?

Reply:
I found this as well - I will open a PR to fix this.